pre-processor-scripts/populate-search-fields.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+/* Since mongo does not support ES6 we can not lint with our current eslint config */
+
+String.prototype.removePunctuation = function () {
```

Comment:
What is the rationale for extending `String`'s prototype? I was under the impression it was a bad idea.